Example/BitmovinComScoreAnalytics_Example_tvOS/ViewController.swift

```diff
@@ -8,14 +8,77 @@
 
 import UIKit
 import BitmovinComScoreAnalytics
+import BitmovinPlayer
+import ComScore
 
 class ViewController: UIViewController {
-
+    var bitmovinPlayer: BitmovinPlayer?
+    @IBOutlet var playerView: UIView!
+    var comScoreStreamingAnalytics: ComScoreStreamingAnalytics?
+    
+    var bitmovinPlayerView: PlayerView?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+        createPlayer()
     }
-
-
+    
+    func createPlayer() {
+        
+        // Define needed resources
+        guard let streamUrl = URL(string: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8") else {
+            return
+        }
+        
+        // Create player configuration
+        let config = PlayerConfiguration()
+        config.playbackConfiguration.isAutoplayEnabled = true
+                
+        do {
+            try config.setSourceItem(url: streamUrl)
+            
+            // Create player based on player configuration
+            let player = BitmovinPlayer(configuration: config)
+            self.bitmovinPlayer = player
+            
+            self.bitmovinPlayer?.add(listener: self)
+            // Create a Comscore Configuration
+            let comScoreMetadata: ComScoreMetadata = ComScoreMetadata(mediaType: .longFormOnDemand, publisherBrandName: "ABC", programTitle: "Modern Family", episodeTitle: "Rash Decisions", episodeSeasonNumber: "1", episodeNumber: "2", contentGenre: "Comedy", stationTitle: "Hulu", completeEpisode: true)
```

Comment:
Change to multiline. Each property in a new line (starting with `publisherBrandName`) aligned with the first one (`mediaType`).